vowpalwabbit/core/include/vw/core/api_status.h

```diff
@@ -260,3 +260,15 @@ int report_error(status_builder& sb, const First& first, const Rest&... rest)
     return sb << VW::experimental::error_code::code##_s
 
 #endif  // RETURN_ERROR_LS
+
```

Comment:
Should I modify RETURN_ERROR_LS based on **status** being nullptr or not (similar to REPORT_ERROR) so that we can optimize it if status is nullptr?